fix(i18n): better translation on link to inventory

jniles opened the pull request:
Gives a better translation for the Articles in Stock link to the inventory.

Closes #4706.

jniles committed:
fix(i18n): better translation on link to inventory

Gives a better translation for the Articles in Stock link to the
inventory.

Closes #4706.

jmcameron commented on client/src/i18n/en/stock.json:
```diff
@@ -135,7 +135,6 @@
-      "EXIT_DEPOTS"     : "Stock exit to depots",
       "EXIT_SERVICE"   : "Stock exit to service",
       "EXIT_SERVICES"   : "Stock exit to services",
       "EXIT_LOSS"      : "Stock Exit - Loss"
```
I noticed that this one is a bit different format from the previous ones.  On the Stock Exit report, they are all presented at the same level, etc, so it looks a bit odd for this one to be different.

jmcameron replied:
The French version is consitent...

jmcameron reviewed (approved):
These changes look good.  I would suggest updating the "Stock Exit - Loss" one (English) to match the rest.   Actually they call could match EXIT_LOSS; they just should be consistent.

jniles committed:
chore(i18n): consistent translations between FR/EN

Makes the translation for stock loss consistent internally and between
languages.
